background/background.js

```diff
@@ -289,32 +289,53 @@ const LivemarkUpdater = {
 
 const ContextMenu = {
   async init() {
-    const createReloadItem = async () => {
+    const getUnreadChildren = async (id) => {
+      const p = await Settings.getReadPrefix();
+      const feed = [...await browser.bookmarks.getChildren(id)]
```

Comment:
I think instead of this complicated filter expression you could use code like here: https://github.com/nt1m/livemarks/blob/0e7d1ecdc6e71e79e7707d8ef0ef86f3c818ba59/background/background.js#L150